packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -631,6 +631,7 @@ export default class Meeting extends StatelessWebexPlugin {
   turnDiscoverySkippedReason: TurnDiscoverySkipReason;
   turnServerUsed: boolean;
   areVoiceaEventsSetup = false;
+  isMoveTo = false;
```

Comment:
maybe `isMoveToInProgress` would be a better name?

Reply:
Done